tests/db_migrator_input/config_db/reclaiming-buffer-warmreboot-expected.json

```diff
@@ -2044,6 +2044,6 @@
         "admin_status": "up"
     },
     "VERSIONS|DATABASE": {
-        "VERSION": "version_3_0_5"
+        "VERSION": "version_3_0_3"
```

Comment:
UT cases are missing on this branch, is that intentional?

Reply:
No, didn't notice it wasn't included will fix in few minutes.
Thanks for the catch.

Reply:
UT added.